app/matplotlib.ipynb

```diff
@@ -47,21 +47,25 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
    "metadata": {
     "collapsed": true,
     "ExecuteTime": {
-     "end_time": "2023-07-07T12:36:31.669207Z",
-     "start_time": "2023-07-07T12:36:31.661449Z"
+     "end_time": "2024-11-08T14:50:24.564617Z",
+     "start_time": "2024-11-08T14:50:24.562494Z"
     }
    },
-   "outputs": [],
    "source": [
     "years = [2015, 2016, 2017, 2018, 2019]\n",
     "population = [100, 120, 150, 180, 200]\n",
     "\n",
-    "# write your code here"
-   ]
+    "plt.plot(years, population)\n",
+    "plt.title(\"World Population Over Time\")\n",
+    "\n",
+    "plt.xlabel(\"Year\")\n",
+    "plt.ylabel(\"Population (Billions)\")\n"
```

Comment:
You need to add `plt.show()` at the end of this cell to display the line plot as per the task requirements.